sormas-api/src/main/java/de/symeda/sormas/api/document/DocumentDto.java

```diff
@@ -104,4 +107,14 @@ public DocumentRelatedEntityType getRelatedEntityType() {
 	public void setRelatedEntityType(DocumentRelatedEntityType relatedEntityType) {
 		this.relatedEntityType = relatedEntityType;
 	}
+
+	public boolean getFileTypeNotAllowed() { return fileTypeNotAllowed; }
```

Comment:
I don't get why these properties have been added. When the file does not have a valid type, the document should not be stored at all, so having this in the dto does not make sense to me.

Reply:
fixed